features/adres.feature

```diff
@@ -95,17 +100,16 @@ Functionaliteit: Adresvelden vullen
     - Wanneer de woonplaats geen waarde heeft, wordt de omschrijving van de gemeente van inschrijving als woonplaats gebruikt
 
     Abstract Scenario: adresregel2 voor een binnenlandse verblijfplaats met <omschrijving>
-      Gegeven het systeem heeft een persoon met de volgende gegevens
+    Gegeven de persoon met burgerservicenummer '<burgerservicenummer>' heeft de volgende 'verblijfplaats' gegevens
```

Comment:
Wijzig
`Abstract Scenario: adresregel2 voor een binnenlandse verblijfplaats met <omschrijving> `
in
`Abstract Scenario: adresregel2 voor een binnenlandse verblijfplaats <omschrijving>`

en vul de 1e kolom van de voorbeeldtabel als volgt
```
| omschrijving                               |
| met woonplaatsnaam anders dan gemeentenaam |
| zonder woonplaatsnaam                      |
```

Reply:
Aangepast